rest/route/github.go

```diff
@@ -542,10 +543,12 @@ func (gh *githubHookApi) handleComment(ctx context.Context, event *github.IssueC
 		return nil
 	}
 
-	if triggerPatch, callerType := triggersPatch(commentBody); triggerPatch {
+	if triggerPatch := triggersPatch(commentBody); triggerPatch {
+		callerType := parsePRCommentForCaller(commentBody)
+		alias := parsePRcommentForAlias(commentBody)
```

Comment:
from the help text, it sounds like we should only be using this alias if the comment is `evergreen patch` as opposed to `evergreen retry` (which given the language, does also seem to make sense). Should this only read from the comment if its the patch comment then?